### PR TITLE
fix: build "full" gettext

### PR DESCRIPTION
**Pull request by smira**
Some programs need `autopoint` for `autoreconf` to work correctly.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

File: squashfs-tools/pkg.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ dependencies:
   - stage: patch
 steps:
   - sources:
-      - url: https://downloads.sourceforge.net/project/squashfs/squashfs/squashfs4.3/squashfs4.3.tar.gz
+      - url: https://mirror.us.leaseweb.net/slackware/slackware64-14.2/source/ap/squashfs-tools/squashfs4.3.tar.gz
```
Comment (smira) [inline squashfs-tools/pkg.yaml:9]: sourceforge is no longer working correctly :(

**Comment by rsmitty** [comment]
Obviously not for this PR, but we should really figure out how to mirror these things ourselves. Relying on these upstream hosts seems dangerous in the longer term.

**Comment by smira** [comment]
> Obviously not for this PR, but we should really figure out how to mirror these things ourselves. Relying on these upstream hosts seems dangerous in the longer term.

fully agreee... on other hand as we have checksums, what is good is that we can jump between mirrors and still have confidence that we get right bits back

(that's how I see sourceforge doesn't work properly, it returns something which doesn't match the checksum anymore).

**Pull request merged**